internal/proto/controller/servers/v1/servers.proto

```diff
@@ -43,3 +43,43 @@ message Server {
 message TagValues {
   repeated string values = 10;
 }
+
+message ServerController {
+  // Private ID of the resource
+  string private_id = 10;
+
+  // Address for the controller
+  string address = 20;
+
+  // Description of the resource (optional)
+  // @inject_tag: `gorm:"default:null"`
+  string description = 30;
+
+  // First seen time from the RDBMS
+  storage.timestamp.v1.Timestamp create_time = 40;
+
+  // Last time there was an update
+  storage.timestamp.v1.Timestamp update_time = 50;
+}
+
+message ServerWorkerStatus {
+  // Public ID of the worker
+  // @inject_tag: `gorm:"primary_key"`
+  string public_id = 10;
```

Comment:
How would the controller calculate the worker public id? Would it try to match an address during the `workerServiceServer.Status()` call?

Reply:
Is that an extra db lookup we otherwise wouldn't have to perform if it's included? So for not-the-first-status-request if the response contains the worker's ID could it include that and eliminate a db lookup later?